src/components/Auth/Filters.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setFilterBy: (object: string) => void;
+  filterBy: string;
+  todos: Todo[] | null;
+  deleteTodo: () => void
+  completedTodos: Todo[];
+};
+
+export const Filter: React.FC<Props> = ({
+  filterBy,
+  todos,
+  setFilterBy,
+  deleteTodo,
+  completedTodos,
+}) => {
+  const handleChangeFilterBy = (filteredBy: string) => {
+    setFilterBy(filteredBy);
+  };
+
+  const filterByParam = (param: string) => {
+    return filterBy === param;
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${todos?.length} items left`}
+
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classNames('filter__link',
+            { selected: filterByParam('all') })}
+          onClick={() => handleChangeFilterBy('all')}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={classNames('filter__link',
+            { selected: filterByParam('active') })}
+          onClick={() => handleChangeFilterBy('active')}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={classNames('filter__link',
+            { selected: filterByParam('completed') })}
+          onClick={() => handleChangeFilterBy('completed')}
+        >
+          Completed
+        </a>
+      </nav>
+
+      <button
+        data-cy="ClearCompletedButton"
+        type="button"
+        className="todoapp__clear-completed"
+        onClick={deleteTodo}
+      >
+        {completedTodos.length > 0 && ('Clear completed')}
```

Comment:
Check how this button works on [the working example](https://mate-academy.github.io/react_todo-app-with-api/) and yours.
<img width="904" alt="Screenshot 2022-10-10 at 19 59 08" src="https://user-images.githubusercontent.com/17098736/194918273-358997cc-2859-4833-a3a8-6eb779931958.png">